test/firtool/split_verilog.mlir

```diff
@@ -5,7 +5,7 @@
 // RUN: FileCheck %s --check-prefix=VERILOG-PLL < %t/pll.v
 
 sv.verbatim "// I'm everywhere"
-sv.ifdef.procedural "VERILATOR" {
+sv.ifdef "VERILATOR" {
```

Comment:
Do we want to treat MLIR modules as procedural? I think that makes sense, but it's a policy decision.

Reply:
It is a bit weird either way, I think that treating the module as a graph region is reasonable.